mnefun/_mnefun.py

```diff
@@ -3650,3 +3686,54 @@ def extract_roi(stc, src, label=None, thresh=0.5):
     func_label = mne.Label(verts, hemi=hemi, subject=stc.subject)
     func_label = func_label.fill(src)
     return func_label, max_vert, max_vidx, max_tidx
+
+
+def do_mf_autobad(p, r, prebad_file):
+    t0 = time.time()
+    tmp_name = op.join(op.dirname(r), 'temp_%s_raw.fif' % t0)
+    tmp_raw = filter_chpi(read_raw_fif(r, allow_maxshield='yes',
+                                       preload=True),
+                          include_line=True)
```

Comment:
Why this step? Is it necessary? I think back in the day when we did this before with shell scripts we did not do cHPI filtering, or if we did, we let MaxFilter do it while processing the data.

Reply:
I wasn’t aware if Maxfilter does cHPI filtering. If so then we can leave the step out.

Reply:
I think it does by default. Most of the time people I've seen people use `autobad` they don't manually remove cHPI themselves before doing it